include/mc_rbdyn/lipm_stabilizer/StabilizerConfiguration.h

```diff
@@ -238,6 +251,7 @@ struct ExternalWrenchConfiguration
     config("modify_com_error", modifyCoMErr);
     config("modify_zmp_error", modifyZMPErr);
     config("modify_zmp_error_d", modifyZMPErrD);
+    config("exclude_from_dcm_bias_est", excludeFromDCMBiasEst);
```

Comment:
This new field has to be documented in the schema: https://github.com/jrl-umi3218/mc_rtc/blob/master/doc/_data/schemas/common/StabilizerConfiguration.json

Reply:
Done ! Thank you